test/test_cuda.py

```diff
@@ -377,6 +377,8 @@ def test_out_of_memory(self):
         tensor.fill_(1)
         self.assertTrue((tensor == 1).all())
 
+
+    @unittest.skipIf(TEST_CUDAMALLOCASYNC, "Segmentation fault (core dumped)")
```

Comment:
Did you confirm these tests didn't run previously in CI?  (as opposed to they were passing in CI but fail/segfault locally)?